projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1359,32 +1371,40 @@ public function process_submission() {
 		 */
 		do_action( 'grunion_after_feedback_post_inserted', $post_id, $this, $is_spam, $entry_values );
 
+		$title   = apply_filters( 'jetpack_forms_response_email_title', __( 'You got a new response!', 'jetpack-forms' ) );
 		$message = self::get_compiled_form_for_email( $post_id, $this );
 
-		array_push(
-			$message,
-			'<br />',
-			'<hr />',
-			__( 'Time:', 'jetpack-forms' ) . ' ' . $time . '<br />',
-			__( 'IP Address:', 'jetpack-forms' ) . ' ' . $comment_author_IP . '<br />', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
-			__( 'Contact Form URL:', 'jetpack-forms' ) . ' ' . $url . '<br />'
-		);
-
 		if ( is_user_logged_in() ) {
-			array_push(
-				$message,
-				sprintf(
-					// translators: the the name of the site.
-					'<p>' . __( 'Sent by a verified %s user.', 'jetpack-forms' ) . '</p>',
-					isset( $GLOBALS['current_site']->site_name ) && $GLOBALS['current_site']->site_name ?
-						$GLOBALS['current_site']->site_name : '"' . get_option( 'blogname' ) . '"'
-				)
+			$sent_by_text = sprintf(
+				// translators: the the name of the site.
+				'<br />' . __( 'Sent by a verified %s user.', 'jetpack-forms' ) . '<br />',
+				isset( $GLOBALS['current_site']->site_name ) && $GLOBALS['current_site']->site_name ? $GLOBALS['current_site']->site_name : '"' . get_option( 'blogname' ) . '"'
 			);
 		} else {
-			array_push( $message, '<p>' . __( 'Sent by an unverified visitor to your site.', 'jetpack-forms' ) . '</p>' );
+			$sent_by_text = '<br />' . __( 'Sent by an unverified visitor to your site.', 'jetpack-forms' ) . '<br />';
 		}
 
-		$message = join( '', $message );
+		$footer = join(
+			'',
+			apply_filters(
+				'jetpack_forms_response_email_footer',
+				array(
+					'<br />',
+					'<hr />',
+					'<span style="font-size: 12px">',
+					__( 'Time:', 'jetpack-forms' ) . ' ' . $time . '<br />',
+					__( 'IP Address:', 'jetpack-forms' ) . ' ' . $comment_author_IP . '<br />', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+					__( 'Source URL:', 'jetpack-forms' ) . ' ' . $url . '<br />',
+					$sent_by_text,
+					'</span>',
+					'<hr />',
+				)
+			)
+		);
+
+		$response_link = admin_url( 'edit.php?post_type=feedback' );
+
+		$message = join( '', apply_filters( 'jetpack_forms_response_email_message', $message ) );
```

Comment:
Do we need this one, given that we have the existing `contact_form_message` just below? If the intent here is to give more flexibility to folks, maybe we could add the array before it's turned into a string as the second parameter of the existing `contact_form_message` filter?

Reply:
Good idea. Thanks.